### PR TITLE
Updates and edits to the press page

### PR DESCRIPTION
**Pull request by awfrancisco**
Edited all sections for style and format. 
Added additional accomplishments 
Removed reference to unfinished blog post.

### CODE REVIEW

File: _press/how-is-18F-funded.md [hunk 1]
```diff
@@ -7,8 +7,7 @@ layout: bare
 
 * GSA funds 18F through the [Acquisition Services Fund][1] (ASF), which allows for investment in the development and delivery of products and tools that will be used by other agencies on a reimbursable basis.
 
-* The Economy Act allows 18F to be reimbursed for services provided to other agencies. Once an agreement is in place, we bill on a quarterly basis through [IPACS][2]. [We’ve written more about how our billing options work on our blog.][3]
```
Comment (gboone) [inline _press/how-is-18F-funded.md]: good catch, we should remember to update this if we ever finish writing that post.

**Comment by gboone** [comment]
All the :+1: from me.

File: _press/how-is-18F-funded.md (hunk 1 above)
Comment (awfrancisco) [inline _press/how-is-18F-funded.md]: I'll add that as a note to the How we bill issue. Which is actually on the calendar for 2 weeks from now.

File: _press/18F-consulting.md [hunk 2]
```diff
@@ -3,6 +3,6 @@ title: 18F Delivery and 18F Consulting
 permalink: /press/18F-consulting
 layout: bare
 ---
-[18F Delivery](/delivery) is the larger of our current service offerings. It is a collection of teams comprised of designers, developers, and product managers that work _for_ and _with_ other federal teams at agencies like the Federal Election Commission, the General Services Administration, and the Treasury ([among many others](https://18f.gsa.gov/dashboard/)) where they build tools to improve government services.
+[18F Delivery](/delivery) is the larger of our current service offerings. It's a collection of teams comprised of designers, developers, and product managers that work _for_ and _with_ other federal teams at agencies like the Federal Election Commission, the General Services Administration, and the Department of the Treasury ([among many others](https://18f.gsa.gov/dashboard/)) where they build tools to improve government services.
```
Comment (quepol) [inline _press/18F-consulting.md]: @gboone @awfrancisco I'd change "where **they** build tools" to "where **we** build tools."

**Comment by awfrancisco** [comment]
Agreed on "where we build tools."

**Comment by awfrancisco** [comment]
I think we can still merge this and then resolve any duplication in a new PR.

**Comment by gboone** [comment]
I'll push up a commit to fix the "where we build" thing :point_up: and then merge.

**Comment by gboone** [comment]
:mouse: :boom: let's check it out on staging and see about fixing up any duplication

**Comment by awfrancisco** [comment]
Checking now.

**Pull request merged**